### ISSUE TITLE
Error in dots_list(...) : attempt to apply non-function

### ISSUE THREAD

**Issue by ndamond** [dynverse/dynguidelines#52]
Hi, I'm getting the above error message when trying to run the `guidelines_shiny()` function.
Any help appreciated

**Example**
```
library(dyno)
dataset <- example_dataset
guidelines_shiny(dataset = dataset)
```
Traceback
```
Error in dots_list(...) : attempt to apply non-function
10. 	 dots_list(...)
9. 	 tags$p("Disconnected trajectories are trajectories which are not connected", tags$im(src = "img/disconnected_example.png"))
8. 	 eval(lhs, parent, parent)
7. 	 eval(lhs, parent, parent)
6. 	 list(list(question_id = "multiple_disconnected", modifier = multiple_disconnected_modifier, type = "radiobuttons", choices = c(Yes = TRUE, `I don't know` = TRUE, No = FALSE), modifier = function(data, answer = NULL) { }, activeIf = "true", label = "Do you expect multiple disconnected trajectories in the data?", ...
5. 	 get_questions()
4. 	 answer_questions(dataset = dataset)
3. 	 shiny_server(answers = answers)
2. 	 shiny::shinyApp(shiny_ui(), shiny_server(answers = answers))
1. 	 guidelines_shiny(dataset = dataset)
```

The shiny app starts (but doesn't work) with:
`guidelines_shiny(dataset = dataset, answers = NULL)`
```
Listening on http://0.0.0.0:4960
Warning: Error in dots_list: attempt to apply non-function
  54: dots_list
  53: tags$p
  49: get_questions
  48: server
Error in dots_list(...) : attempt to apply non-function
```
  
`sessionInfo()`
```
R version 4.0.1 (2020-06-06)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.4 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.7.1
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.7.1

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8        LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8   
 [6] LC_MESSAGES=C.UTF-8    LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C           LC_TELEPHONE=C        
[11] LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] dyno_0.1.2            dynwrap_1.2.1         dynplot_1.0.2.9000    dynmethods_1.0.5      dynguidelines_1.0.0  
[6] dynfeature_1.0.0.9000

loaded via a namespace (and not attached):
  [1] dynutils_1.0.5       RColorBrewer_1.1-2   tools_4.0.1          backports_1.1.8      utf8_1.1.4          
  [6] R6_2.4.1             irlba_2.3.3          rpart_4.1-15         Hmisc_4.4-0          colorspace_1.4-1    
 [11] nnet_7.3-14          tidyselect_1.1.0     carrier_0.1.0        gridExtra_2.3        processx_3.4.2      
 [16] proxyC_0.1.5         compiler_4.0.1       cli_2.0.2            htmlTable_2.0.0      scales_1.1.1        
 [21] checkmate_2.0.0      readr_1.3.1          stringr_1.4.0        digest_0.6.25        foreign_0.8-80      
 [26] base64enc_0.1-3      jpeg_0.1-8.1         pkgconfig_2.0.3      htmltools_0.5.0      GA_3.2              
 [31] fastmap_1.0.1        rje_1.10.16          htmlwidgets_1.5.1    rlang_0.4.6          rstudioapi_0.11     
 [36] shiny_1.4.0.2        generics_0.0.2       farver_2.0.3         acepack_1.4.1        dplyr_1.0.0         
 [41] magrittr_1.5         Formula_1.2-3        patchwork_1.0.1.9000 Matrix_1.2-18        fansi_0.4.1         
 [46] Rcpp_1.0.4.6         dyndimred_1.0.3.9000 munsell_0.5.0        viridis_0.5.1        lifecycle_0.2.0     
 [51] stringi_1.4.6        yaml_2.2.1           ggraph_2.0.3         MASS_7.3-51.6        plyr_1.8.6          
 [56] grid_4.0.1           promises_1.1.1       ggrepel_0.8.2        crayon_1.3.4         lattice_0.20-41     
 [61] cowplot_1.0.0        graphlayouts_0.7.0   splines_4.0.1        hms_0.5.3            knitr_1.29          
 [66] ps_1.3.3             pillar_1.4.4         igraph_1.2.5         ranger_0.12.1        babelwhale_1.0.1    
 [71] codetools_0.2-16     reshape2_1.4.4       glue_1.4.1           latticeExtra_0.6-29  data.table_1.12.8   
 [76] remotes_2.1.1        RcppParallel_5.0.1   foreach_1.5.0        png_0.1-7            vctrs_0.3.1         
 [81] dynparam_1.0.0       tweenr_1.0.1         httpuv_1.5.4         testthat_2.3.2       gtable_0.3.0        
 [86] purrr_0.3.4          polyclip_1.10-0      tidyr_1.1.0          assertthat_0.2.1     ggplot2_3.3.2       
 [91] xfun_0.15            ggforce_0.3.1        lmds_0.1.0           mime_0.9             xtable_1.8-4        
 [96] tidygraph_1.2.0      later_1.1.0.1        survival_3.1-12      viridisLite_0.3.0    tibble_3.0.1        
[101] iterators_1.0.12     cluster_2.1.0        ellipsis_0.3.1   
```

**Comment by rcannood** [comment]
Hello @ndamond !

Thanks for reporting this issue. I have fixed it in 793702a312729737bdae3da925bdfb6a9c27e26a, which should now be available on the master branch. Could you try updating dynguidelines to version 1.0.1 to see if this indeed solves your problem?

Kind regards,
Robrecht

**Comment by ndamond** [comment]
Thanks a lot @rcannood, it works now !

Cheers,
Nicolas
